webassembly-language-support.md

```diff
@@ -18,50 +18,67 @@ This guide tracks support for compiling a language to WebAssembly. It is organiz
+| [Scala (JVM)][Scala]      | ✅    | ✅      | ✅   |          |
+| [Scala (native)][Scala]   | ⏳    | ❌      | ❌   | ❌       | 
+| [Go][Go]                  | ✅    | ✅      | ✅   | ✅       |
+| [PowerShell][PowerShell]  | ❌    | ❌      | ❌   | ❌       |
+| [Kotlin (JVM)][Kotlin]    | ✅    | ✅      | ✅   | ❌       |
+| [Kotlin (Native)][Kotlin] | ⏳    | ✅      | ⏳   | ❌       |
+| [Rust][Rust]              | ✅    | ✅      | ✅   | ✅       |
+| [Dart][Dart]              | ❌    | ⏳      | ❌   | ❌       |
+
+* _Core_ means there is an implementation of WebAssembly 1.0
+* _Browser_ means there is at least one browser implementation
+* _Wasi_ means that the language supports at least WASI Preview 1
+* _Spin SDK_ indicates there is a Spin SDK for the language
+
+Anything with WASI or Spin SDK support runs on Fermyon Cloud, Spin, and Fermyon Platform.
 
 ## WebAssembly Specific Languages
 
-| Language                  | Browser | CLI | WASI | Notes |
-| ------------------------- | ------- | --- | ---- | ----- |
-| [AssemblyScript][AssemblyScript] | ✅  | ✅ | ✅ | |
-| [Grain][Grain]                   | ✅  | ✅ | ✅ | |
-| [Motoko][Motoko]                 | ✅  | ✅ | ✅ | |
+| Language                         | Browser | CLI | WASI | Spin SDK |
+| -------------------------------- | ------- | --- | ---- | -------- |
+| [AssemblyScript][AssemblyScript] | ✅      | ✅  | ✅   | ❌       |
+| [Grain][Grain]                   | ✅      | ✅  | ✅   | ❌       |
+| [Motoko][Motoko]                 | ✅      | ✅  | ✅   | ❌       |
+
+* _Browser_ means there is at least one browser implementation
+* _CLI_ means the language has a CLI runtime mode
+* _Spin SDK_ indicates there is a Spin SDK for the language
 
 ## Other Notable Languages
 
 These languages enjoy broad use (though perhaps not in the top 20) and have at least some degree of WebAssembly Support
 
-| Language                  | Browser | CLI | WASI | Notes |
-| ------------------------- | ------- | --- | ---- | ----- |
-| [COBOL][Cobol]            |⏳ | ✅ | ⏳ |  |
-| [Erlang (BEAM)][Erlang]   | ⏳ | ⏳ | ⏳ |  |
-| [Haskell][Haskell]        | ✅  | ✅ | ✅ | |
-| [Lisp][Lisp]              | ⏳ | ⏳ | ⏳ |  |
-| [Lua][Lua]                | ✅ | ❌ | ❌ |  |
-| [Perl][Perl]              | ✅ | ❌ | ❌ |  |
-| [Zig][Zig]                | ✅  | ✅ | ✅ | |
+| Language                  | Browser | CLI | WASI | Spin SDK |
+| ------------------------- | ------- | --- | ---- | -------- |
+| [COBOL][Cobol]            | ⏳      | ✅  | ⏳   | ❌       |
+| [Erlang (BEAM)][Erlang]   | ⏳      | ⏳  | ⏳   | ❌       |
+| [Haskell][Haskell]        | ✅      | ✅  | ✅   | ❌       |
+| [Lisp][Lisp]              | ⏳      | ⏳  | ⏳   | ❌       |
+| [Lua][Lua]                | ✅      | ❌  | ❌   | ❌       |
+| [Perl][Perl]              | ✅      | ❌  | ❌   | ❌       |
+| [Zig][Zig]                | ✅      | ✅  | ✅   | ❌       |
+
+* _Browser_ means there is at least one browser implementation
+* _CLI_ means the language has a CLI runtime mode
```

Comment:
Should we also include `* _Wasi_ means that the language supports at least WASI Preview 1` for completeness?

Reply:
Done